portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,12 +34,19 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(COMMENTS);
+    String json = convertArrayToJson(COMMENTS);
     response.setContentType("text/html;");
     response.getWriter().println(json);
   }
 
-  private static String convertToJson(String[] comments) {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String body = request.getReader().lines().reduce("", String::concat); // grabs request body
```

Comment:
talked offline on how to convert the response to send parameters instead of parsing body, to be done in future PR

Reply:
updating previous comments, ok to leave as is

Reply:
okidoki